style.css

```diff
@@ -0,0 +1,107 @@
+/* CSS reset */
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+
+/* HTML5 display-role reset for older browsers */
+
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+
+body {
+	line-height: 1;
+}
+
+ol, ul {
+	list-style: none;
+}
+
+blockquote, q {
+	quotes: none;
+}
+
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+.subscribe{
+	margin: 0;
+}
+
+nav{
+	border-bottom: 1px solid #ccc;
+}
+
+.main-nav  {
+	margin-left: auto;
+	margin-right: auto;
+	max-width: 1000px;
+}
+
+.main-nav .logo{
+	float: left;
+	height: 32px;
+	width: 100px;	
+}
+.logo a{
+	padding: 8px 0px 0px 100px;
+	border-right: 1px solid #ccc;
+}
+.main-menu{
+	height: 40px;
+	width: 800px;	
+	margin-left: 300px;
+}
+.main-menu  li{
+	float: left;
+	list-style: none;
+	height: 23px;
+	width: 40px;
+	padding: 14px 14px 0px 14px;	
+	border-left: 1px solid #ccc;
+}
```

Comment:
I set `height` to `38px`, since the borders in the `.main-menu` do no completely touch the bottom borders of `main-nav`.